### DISCUSSION TITLE
Docker which tag to use?

### DISCUSSION THREAD

**Discussion by darrepac** [photoview/photoview#699] [General]
Hi

I am using `image: viktorstrate/photoview:2` but it seems to run 2.3.9 version.  Is there a newer docker image with a more recent release?
Which tag to use for it?

thanks

**Comment by viktorstrate** [comment]
See the documentation https://photoview.github.io/en/docs/installation-docker/#docker-tags-and-versioning

**Comment by darrepac** [comment]
oups miss that.
I used "latest" tag and it is still written 2.3.9 in settings. Is this correct?
